lib/qubes-rpc-multiplexer

```diff
@@ -51,14 +51,5 @@ for DIR in $QREXEC_SERVICE_PATH; do
 done
 IFS=$ifs
 
-if [ -x "$CFG_FILE" ] ; then
-    # shellcheck disable=SC2086
-	exec "$CFG_FILE" ${QREXEC_SERVICE_ARGUMENT}
-	echo "$0: failed to execute handler for" "$1" >&2
```

Comment:
You removed this error message, intentional?

Reply:
Yes; most shells never return from a failed `exec`.

Reply:
I don't think so:
```
$ test -x /tmp/error.sh; echo $?
0
$ exec /tmp/error.sh; echo $?
bash: /tmp/error.sh: /bin/no-such-file: bad interpreter: No such file or directory
bash: /tmp/error.sh: Success
126
$ cat /tmp/error.sh
#!/bin/no-such-file
```